README.md

```diff
@@ -1,2 +1,52 @@
-# Awesome-Book
-Plain JavaScript with Classes to add and remove book from an array of object and using local storage
+# Project Name
```

Comment:
Great job @mrbhi 🤝

- [ ] Kindly customize the readme file correctly. You may remove the second author if this is a solo project, Add a project name, and make sure the GitHub link works perfectly for the authors and directs the user to the right page.

Reply:
Thanks @1GORDON